X10D/src/Collections/DictionaryExtensions.cs

```diff
@@ -187,9 +419,9 @@ public static class DictionaryExtensions
         }
 #endif
 
-        if (dictionary.ContainsKey(key))
+        if (dictionary.TryGetValue(key, out var old))
```

Comment:
And this one. Since the assigned type here is not inherently apparent, explicitly declaring old as TValue is preferred.